tests/components/unifi/test_switch.py

```diff
@@ -384,7 +385,7 @@
     "x_vwirekey": "2dabb7e23b048c88b60123456789",
     "vwire_table": [],
     "dot1x_portctrl_enabled": False,
-    "outlet_overrides": [],
+    "outlet_overrides": [{"index": 1, "name": "Outlet 1", "relay_state": True}],
```

Comment:
Why did you change this?

Reply:
This was changed to make it easier to set up the test parameterized test. On my UP there are overrides already in place, so I figured it was still a representative test.